lib/vanagon/component/source/git.rb

```diff
@@ -5,20 +5,30 @@ class Component
     class Source
       class Git
         include Vanagon::Utilities
-        attr_accessor :url, :ref, :workdir, :version, :cleanup
+        attr_accessor :url, :ref, :workdir, :version, :cleanup, :clone_args
 
         # Constructor for the Git source type
         #
         # @param url [String] url of git repo to use as source
         # @param ref [String] ref to checkout from git repo
         # @param workdir [String] working directory to clone into
-        def initialize(url, ref, workdir)
+        def initialize(url, ref, workdir, clone_args)
```

Comment:
I wanted it to just be nil if it wasnt set. But I can just default it to a empty string